npm/vite-dev-server/client/reactDomClientPlaceholder.js

```diff
@@ -0,0 +1,2 @@
+// Placeholder ...
```

Comment:
Might be nice to describe the intent/purpose of this placeholder. I guess there is a comment in your `react18.ts` Vite plugin so I guess it's documented, but maybe we can just hint to it here `Placeholder for React18 plugin`

Reply:
Will do, thanks!